src/DynamoCoreWpf/ViewModels/Watch3D/Watch3DViewModelBase.cs

```diff
@@ -25,7 +25,6 @@ public class Watch3DViewModelStartupParams
         public IRenderPackageFactory RenderPackageFactory { get; set; }
         public IDynamoViewModel ViewModel { get; set; }
         public INotifyPropertyChanged RenderPackageFactoryViewModel { get;set; }
-        public bool IsActiveAtStart { get; set; }
```

Comment:
This property is not required as we can already get at the pref. setting through the `IPreferences` object.